canopen_tests/launch/robot_control_setup.launch.py

```diff
@@ -0,0 +1,111 @@
+from launch import LaunchDescription
```

Comment:
Again copied from [here](https://github.com/ros- industrial/ros2_canopen/blob/master/canopen_ros2_control/launch/cia402_system.launch.py) and no license presented.